internal/core/accessibility.rs

```diff
@@ -24,6 +30,68 @@ pub enum AccessibleStringProperty {
     ValueStep,
 }
 
+// Defines an accessibility action.
+pub enum AccessibilityAction {
+    Default,
+    Focus,
+    Blur,
+    Collapse,
+    Expand,
+    CustomAction,
+    Decrement,
+    Increment,
+    // FIXME: Can it be currently supported?
+    HideToolTip,
+    // FIXME: Can it be currently supported?
+    ShowToolTip,
+    ReplaceSelectedText(SharedString),
+    ScrollBackward,
+    ScrollDown,
+    ScrollForward,
+    ScrollLeft,
+    ScrollRight,
+    ScrollUp,
+    ScrollIntoView,
+    ScrollToPoint(LogicalPoint),
+    SetScrollOffset(LogicalVector),
+    SetTextSelection(Option<core::ops::Range<i32>>),
+    // FIXME: Do we need this?
+    SetSequentialFocusNavigationStartingPoint,
+    SetValue(f64),
+    // FIXME: Can it be currently supported?
+    ShowContextMenu,
+}
+
+bitflags! {
+    // Define a accessibility actions that currently supported by Slint.
+    pub struct SupportedAccessibilityAction: u32 {
+        const Default = 0b00000001;
+        const Focus = 0b00000010;
+        const Blur = 0b00000011;
```

Comment:
I thought the const in a bitflag can only have a single bit set.

maybe the `1 << X` notation would be better

Reply:
I changed it, is that what you mean?

Reply:
@ogoffart ?

Reply:
Yes, this is what i meant.